habitat_baselines/common/tensorboard_utils.py

```diff
@@ -1,8 +1,30 @@
-from typing import Optional, Union
+from typing import Union
 
 import numpy as np
 import torch
-from torch.utils.tensorboard import SummaryWriter
+
+
+class DummyWriter:
```

Comment:
We can use `SummaryWriter = None` instead of using a `DummyWriter` here and handle the logic later.

Reply:
I think I'll create a separate PR for this. This PR did not change any of the tensorboard-specific logic and inherited directly from #127. I do agree that DummyWriter is a bit clunky and if we properly check `None` when appropriate we can get rid of the wrapper altogether.

Reply:
Can you create a TODO for above.